nnoir/test/test_run.py

```diff
@@ -456,3 +456,11 @@ def test_Unpooling2D():
         outh=out_h,
         outw=out_w,
     )
+
+
+def test_Exp():
```

Comment:
plz move `test_Exp` to between `test_ELU` and `test_Gemm` to keep lexicographic order.